tests/migration_test.go

```diff
@@ -114,10 +114,10 @@ var _ = Describe("Migrations", func() {
 
 	Describe("Starting a VirtualMachineInstance ", func() {
 		Context("with an Alpine read only disk", func() {
-			It("should be successfully migrated multiple times", func() {
+			It("should be successfully migrated multiple times with cloud-init disk", func() {
 
 				vmi := tests.NewRandomVMIWithEphemeralDisk(tests.ContainerDiskFor(tests.ContainerDiskAlpine))
```

Comment:
maybe we shoudl consider changing this from alpine to cirros since alpine doesn't interpret cloud-init noCloud.

Reply:
Yep, done.